elastic-search-support/src/main/java/uk/gov/hmcts/ccd/definition/store/elastic/ElasticDefinitionImportListener.java

```diff
@@ -48,22 +48,34 @@ public ElasticDefinitionImportListener(CcdElasticSearchProperties config, CaseMa
     public void initialiseElasticSearch(List<CaseTypeEntity> caseTypes) {
         HighLevelCCDElasticClient elasticClient = null;
         String caseMapping = null;
+        String eventCaseMapping = null;
+
         CaseTypeEntity currentCaseType = null;
         try {
             elasticClient = clientFactory.getObject();
```

Comment:
This code is a generic prototype to create a new EventCases index and mappings. There is a lot of common code with create Cases index and mappings.  The concrete implementation should create common  generic code and add more test coverage EventCases index, mappings and alias